terraform/modules/network-validator/variables.tf

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+variable "wls_subnet_id" {
+  type        = string
+  description = "The OCID of the subnet where WebLogic VMs will be created"
+}
+
+variable "bastion_subnet_id" {
+  type        = string
+  description = "The OCID of the subnet for the bastion instance"
+}
+
+variable "bastion_ip" {
+  type        = string
+  description = "Bastion Host IP of the existing bastion"
+}
+
+variable "lb_subnet_1_id" {
+  type        = string
+  description = "The OCID of a regional or AD-specific subnet for primary load balancer"
+}
+
+variable "lb_subnet_2_id" {
+  type        = string
+  description = "The OCID of a regional or AD-specific subnet for secondary load balancer"
```

Comment:
Strictly speaking, this variable should never take the ocid of a regional subnet. It is used only for AD-specific subnets

Reply:
Fixed it.